data/Santini_2016/metadata.yml

```diff
@@ -0,0 +1,349 @@
+    and used the software Image J 1.48v (National Institutes of Health, USA) to measure
+    xylem traits. We determined the minor (a) and major (b) axis diameters of xylem
+    vessels to calculate vessel area (VA). We also determined vessel density (VD)
+    (Eq. 2) to calculate conductive area (CA) of each species (Lewis 1992) (see manuscripts
+    for equations).
+- var_in: Ks
+  unit_in: kg/mm/MPa/s
+  trait_name: sapwood_specific_conductivity_theoretical
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 9
+  methods: Branches were collected and stored in sealed vials containing 50% ethanol.
+    Cross sections of each branch were fixed in formalin-acetic acid-alcohol (FAA)
+    for 7 days and then placed in 70 % ethanol for 2 days; we repeated this washing
+    process twice. Micro-sections of all species with the exception of E. vespertilio
+    were prepared with a sledge Leica SM2010R microtome. Micro-sections of E. vespertilio
+    were pre-treated with ethanol (8 h), xylene (4 h) and finally embedded in paraffin
+    with a Shandon Histocentre 3 histology-embedding centre (Thermo Fisher Scientific,
+    Australia) before cutting with a Microm HM325 rotatory microtome (Thermo Fisher
+    Scientific, Australia). Micro-sections of all species were double stained with
+    safranin-alcian blue. We photographed the micro-sections with a Leica DM750 microscope
+    and used the software Image J 1.48v (National Institutes of Health, USA) to measure
+    xylem traits. We determined the minor (a) and major (b) axis diameters of xylem
+    vessels to calculate vessel area (VA) following Eq. (1). We also determined vessel
+    density (VD) (Eq. 2) to calculate conductive area (CA) of each species as in Eq.
+    (3) (Lewis 1992)
+- var_in: root wood density (g cm-3)
+  unit_in: g/cm3
+  trait_name: root_wood_density
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: root wood density replicates
+  methods: Root material collected for assessment of wood density was stored in paper
+    bags and kept cool for laboratory analysis. Wood density of roots was measured
+    in 4-cm lengths of sapwood. Sample numbers of individual roots (i.e. one root
+    sample per plant) for assessment of wood density were as follows, Erythrina vespertilio
+    (n=5), Acacia aneura (n=5), Hakea sp. (n=5), Psydrax latifolia (n=5), Corymbia
+    opaca (n=3) and Eucalyptus camaldulensis (n=5). Root sapwood segments were weighed
+    after drying to a constant mass. Wood density was calculated as the ratio of dry
+    weight (5 days at 60°C) to volume of water displaced by the fully hydrated sapwood
+    segments (i.e., after they were submerged in water for two days; see Santini et
+    al. 2016).
+- var_in: stem Sapwood density (g cm-3)
+  unit_in: g/cm3
+  trait_name: wood_density
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 6
```

Comment:
Just wondering where `replicates: 6` came from, unless this is something the author mentioned.

Reply:
I don't either - but I'd updated from raw data, because I now know actual # of individuals